src/index.tsx

```diff
@@ -12,36 +12,57 @@ import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
-Sentry.init({
-  ...(window.location.hostname === 'localhost'
-    ? {}
-    : { dsn: getConfig().REACT_APP_SENTRY_DSN_URL }),
-  attachStacktrace: true,
-  denyUrls: ['alchemy.com'],
-  integrations: [
-    // Sentry.browserTracingIntegration(),
-    // Sentry.replayIntegration(),
-    Sentry.httpClientIntegration({
-      failedRequestStatusCodes: [
-        [400, 428],
-        // i.e no 429s
-        [430, 599],
-      ],
-    }),
-    Sentry.browserApiErrorsIntegration(),
-    Sentry.breadcrumbsIntegration(),
-    Sentry.globalHandlersIntegration(),
-    Sentry.httpContextIntegration(),
-  ],
-  beforeSend(event) {
-    // https://github.com/getsentry/sentry-javascript/issues/8353 / https://forum.sentry.io/t/turn-off-event-grouping/10916/3
-    event.fingerprint = [(Math.random() * 1000000).toString()]
-    return event
-  },
-  enableTracing: true,
-  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
-  tracePropagationTargets: ['localhost'],
-})
+// Remove this condition to test sentry locally
+if (window.location.hostname !== 'localhost') {
+  const VALID_ENVS = [
+    'localhost',
+    'develop',
+    'release',
+    'app',
+    'private',
+    'yeet',
+    'beard',
+    'juice',
+    'wood',
+    'gome',
+  ] as const
+
+  const environment = (() => {
+    if (window.location.hostname.includes('app')) return 'production'
```

Comment:
`app` is the odd one - it doesn't match its hostname as it should be `production`.
All others *do* reflect their hostname.